### DISCUSSION TITLE
Search dropdown displayed data

### DISCUSSION THREAD

**Discussion by rushattac** [google/timesketch#1853] [Q&A, quick solutions, support] [answered]
Is it on purpose that the search dropdown only shows the top 10 data types and tags based on the count?

**Comment by jaegeral** [comment, marked as answer]
Hey @rushattac  yes this is WAI, curious if you would prefer to see other data in there?

**Reply by rushattac** [reply]
hi @jaegeral  it seems that this has been brought up as a bug in #1978
